docs/quick-open-file.md

```
@@ -0,0 +1,307 @@
+- `testCommandsScopeWithPrefix` — `>` → `.commands`
+- `testFileSearchScopeWithPrefix` — `@`、`@/`、`@~`、`@main` → `.fileSearch`
+- `testSwitcherScopeWithoutPrefix` — 无前缀 → `.switcher`
+- `testCommandsPrefixTakesPriorityOverFileSearch` — `>@` → `.commands`（`>` 优先）
+
+**去重指纹（3 个）:**
+- `testFileSearchPathModeDoesNotUseDedupFingerprint` — path 模式不使用 cross-directory 去重
+- `testFileSearchCrossDirectoryUsesEffectiveQueryAndWorkspaceForDedupFingerprint` — query + workspace root 共同决定去重
+- `testNonFileSearchDoesNotUseFileSearchDedupFingerprint` — 非 file search 不使用该指纹
+
+**Query 提取（5 个）:**
+- `testFileSearchMatchingQueryEmpty` — `@`、`@ ` → `""`
```

markdownlint-cli2 (0.22.1)

[warning] Spaces inside code span elements

(MD038, no-space-in-code)